docs/source/en/training/text2image.mdx

```diff
@@ -155,6 +155,28 @@ python train_text_to_image_flax.py \
 </jax>
 </frameworkcontent>
 
+## Training with better convergence
```

Comment:
Maybe 
```suggestion
## Training with Min-SNR weighting
```

As I think we don't know (yet?) if it'll always lead to better convergence for text2image. But no strong preference :)